packages/core/locales/fr/zod.json

```diff
@@ -13,6 +13,7 @@
       "custom": "Champ invalide",
       "invalid_intersection_types": "Les résultats d'intersection n'ont pas pu être fusionnés",
       "not_multiple_of": "Nombre doit être multiple de {{multipleOf}}",
+      "not_finite": "Number must be finite",
```

Comment:
@ismailajizou Please tell me the translation of this message.

Reply:
The translation is: Nombre doit être fini

Reply:
@ismailajizou Thank you!